app/Http/Controllers/LaravelFormController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\LaravelForm;
+use App\Models\LaravelTechnology;
+
+class LaravelFormController extends Controller
+{
+    public function index()
+    {
+        // laravel_formsテーブルからすべてのデータを取得
+        $forms = LaravelForm::all();
+         $technologies = LaravelTechnology::with('forms')->get();
```

Comment:
インデントがずれているので修正お願いいたします。
```suggestion
        $technologies = LaravelTechnology::with('forms')->get();
```

Reply:
インデントの修正を行ったので、確認をお願いします。